### PR TITLE
Add blogs for airflow DAGs testing

### PR DESCRIPTION
**Pull request by chandulal**

### CODE REVIEW

File: landing-pages/site/content/en/blog/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+linkedin: "chandu-kavar-627a209b"
+description: "To know how to test the xComs and DAGs end-to-end"
+tags: ["Testing"]
+date: "2020-01-03"
+---
+In [part 1](https://blog.usejournal.com/testing-in-airflow-part-1-dag-validation-tests-dag-definition-tests-and-unit-tests-2aa94970570c) of this blog, I explained DAG validation tests, DAG definition tests, and Unit tests. These tests help to verify the syntax and cyclicity in the DAG, upstream and downstream of each task, the logic of the custom operator and sensor, etc. I would recommend reading the first part of this post.
+
+In this part, I covered the integration and end-to-end pipeline tests.
+
+You can find this blog on the medium [Testing in Airflow Part 2](https://medium.com/@chandukavar/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests-af0555cd1a82).
```
Comment (louisguitton) [inline landing-pages/site/content/en/blog/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests.md]: Just to mention that as a reader, I often ran into the Medium paywall until I converted recently.
For those people who are not paying Medium customers, I think there is a way for authors to get a paywall-free link to their articles. Is it the case here ? 
Else, do you have that article on another (free) platform like for part 1 ?

Comment (chandulal) [reply landing-pages/site/content/en/blog/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests.md]: I made part2 paywall-free on medium as well.

Comment (chandulal) [reply landing-pages/site/content/en/blog/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests.md]: Let me update the free URL.

Comment (chandulal) [reply landing-pages/site/content/en/blog/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests.md]: I added a friend link for the safer side.

File: landing-pages/site/content/en/blog/testing-in-airflow-part-1-dag-validation-tests-dag-definition-tests-and-unit-tests.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+tags: ["Testing"]
+date: "2020-01-03"
+---
+Testing is an integral part of any software system to build confidence and increase the reliability of the system. Recently, I joined Grab and here at Grab, we are using Airflow to create and manage pipelines. But, we were facing issues with Airflow. I had a conversation with my engineering manager and discussed on how we could make Airflow reliable and testable.
+Before Grab, I worked with ThoughtWorks. They are using TDD (Test Driven Development) methodology in almost all the projects. As a matter of fact, everyone is very cognizant about testing. This motivated me to explore testing in Airflow.
+I spent a weekend Googling. But, I didn’t find any good articles on airflow test covering all the aspects, there are only a handful of articles available. I felt it’s worth to write and share a blog on airflow testing.
+
+You can find more details on how to write these tests with examples in [Testing in Airflow Part 1](https://blog.usejournal.com/testing-in-airflow-part-1-dag-validation-tests-dag-definition-tests-and-unit-tests-2aa94970570c?source=friends_link&sk=3e1ec0b1e5fcc1e9c258e08d260c7152).
```
Comment (kaxil) [inline landing-pages/site/content/en/blog/testing-in-airflow-part-1-dag-validation-tests-dag-definition-tests-and-unit-tests.md]: Sorry, the content needs to entirely on the Airflow website.

File: landing-pages/site/content/en/blog/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+linkedin: "chandu-kavar-627a209b"
+description: "To know how to test the xComs and DAGs end-to-end"
+tags: ["Testing"]
+date: "2020-01-03"
+---
+In [part 1](https://blog.usejournal.com/testing-in-airflow-part-1-dag-validation-tests-dag-definition-tests-and-unit-tests-2aa94970570c?source=friends_link&sk=3e1ec0b1e5fcc1e9c258e08d260c7152) of this blog, I explained DAG validation tests, DAG definition tests, and Unit tests. These tests help to verify the syntax and cyclicity in the DAG, upstream and downstream of each task, the logic of the custom operator and sensor, etc. I would recommend reading the first part of this post.
+
+In this part, I covered the integration and end-to-end pipeline tests.
+
+You can find this blog on the medium [Testing in Airflow Part 2](https://medium.com/@chandukavar/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests-af0555cd1a82?source=friends_link&sk=0841c4c3bf786a48e748f0e6c1fd189f).
```
Comment (kaxil) [inline landing-pages/site/content/en/blog/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests.md]: Same here: Sorry, the content needs to entirely on the Airflow website.

Comment (kaxil) [reply landing-pages/site/content/en/blog/testing-in-airflow-part-2-integration-tests-and-end-to-end-pipeline-tests.md]: Btw I saw that there were similar blog posts that were published. I have raised this issue and asked for feedback from the entire community.

Feel free to voice your opinion on the mailing list :)

**Comment by turbaszek** [comment]
Closing due to no response from author

**Pull request closed without merging**